recipes-connectivity/networkmanager/networkmanager_1.22.%.bbappend

```diff
@@ -1,4 +1,4 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}-${PV}:${THISDIR}/files:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}-1.22:${THISDIR}/files:"
```

Comment:
OE supports `-%` in the dir name, so this can stay `${PV}` and the patches dir gets named to ```networkmanager-1.22.%```.

Reply:
Actually I guess it's the same effect as naming it `networkmanager-1.22`, the real benefit would be to set it to `networkmanager-1.%` to handle minor and micro version changes.

Reply:
I think I assumed that did not work...  Thanks!